package/MDAnalysis/analysis/encore/bootstrap.py

```diff
@@ -150,6 +150,6 @@ def get_ensemble_bootstrap_samples(ensemble,
             size=ensemble.trajectory.timeseries().shape[1])
         ensembles.append(
             mda.Universe(ensemble.filename,
-                        ensemble.trajectory.timeseries(format='afc')[:,indices,:],
+                        ensemble.trajectory.timeseries(format='fac')[indices,:,:],
```

Comment:
you can now actually skip the last `:, :]` in the fancy index. Numpy will default to taking the full array for lower dimensions